packages/contracts-bedrock/scripts/storage-snapshot.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+set -e
+
+if ! command -v forge &> /dev/null
+then
+    echo "forge could not be found. Please install forge by running:"
+    echo "curl -L https://foundry.paradigm.xyz | bash"
+    exit
+fi
+
+contracts=(
+  L1CrossDomainMessenger
+  L1StandardBridge
+  L2OutputOracle
+  OptimismPortal
+  DeployerWhitelist
+  GasPriceOracle
+  L1Block
+  L1BlockNumber
+  L2CrossDomainMessenger
+  L2StandardBridge
+  L2ToL1MessagePasser
+  OVM_ETH
+  SequencerFeeVault
+  WETH9
+  ProxyAdmin
+  Proxy
+  L1ChugSplashProxy
+  OptimismMintableERC20
+  OptimismMintableTokenFactory
+)
```

Comment:
This is probably overkill for contracts to list. But I figured it didn't hurt to include more rather than fewer.